werkomgeving/gegeven/index.md

```diff
@@ -5,8 +5,8 @@
 Een gegeven is *een vastgelegde uitspraak over een getypeerde eigenschap van een object*. Kenmerkend aan een gegeven zijn een aantal elementen:
 
 1. Een gegeven is **vastgelegd**: op papier, in steen, in een digitale vorm. In elk van deze vormen van sprake zijn van meer of minder mate van vluchtigheid of persistentie: gegevens in een bericht zullen minder persistent zijn dan gegevens die duurzaam worden opgeslagen in een administratie.
-2. Een gegeven is een **uitspraak**: zo'n uitspraak kan een bewering, stelling of waarneming zijn en zo'n uitspraak kan gedaan worden door een mens of een sensor. Ook kan zo'n uitspraak betrekking hebben op een berekening of afleiding.
-3. Een gegeven gaat over een **object**: het gaat over *iets*: de objecten die relevant zijn in het domein waarover we gegevens verwerken.
+2. Een gegeven gaat over een **object**: het gaat over *iets*: de objecten die relevant zijn in het domein waarover we gegevens verwerken.
+3. Een gegeven is een **uitspraak**: zo'n uitspraak kan een bewering, stelling of waarneming zijn en zo'n uitspraak kan gedaan worden door een mens of een sensor. Ook kan zo'n uitspraak betrekking hebben op een berekening of afleiding.
 4. Een gegeven betreft een **eigenschap** van dat object: het gegeven legt een uitspraak vast over een eigenschap die het object heeft. Dit kan zowel een kenmerk van het object zelf betreffen, als een relatie die een object heeft met een ander object.
 
 > Vijf voorbeelden van een gegeven:
```

Comment:
Kunnen we ook een voorbeeld van een ‘Stelling’ verzinnen?

In onderstaand lijstje is n.m.m.
* de 2e en de 5e een ‘Bewering’ want het zijn geen intrinsieke eigenschapen van Jan en kunnen dus niet waargenomen worden.
* de 3e en de 4e een ‘Waarneming’ want het zijn intrinsieke eigenschappen van Jan en kunnen dus waargenomen worden al geldt dat dan weer niet voor een ‘geboortedatum’ wat denk ik toch een ‘Waarneming’ is.

De vraag is alleen wat de 1e is. Een ‘Bewering’, ‘Waarneming’, ‘Stelling’ of nog iets anders? Wellicht is dit juist een ‘Afleiding’? In dat geval moet punt 3 hierboven daarmee uitgebreid worden.

Reply:
Toegevoegd